.github/workflows/ci-p4tools.yml

```diff
@@ -27,12 +27,12 @@ jobs:
     - name: ccache
       uses: hendrikmuhs/ccache-action@v1
       with:
-        key: test-tools-${{ matrix.unified }}-${{ runner.os }}
+        key: test-tools-${{ runner.os }}
         max-size: 1000M
 
     - name: Build (Ubuntu Linux)
       run: |
-        docker build -t p4c --build-arg IMAGE_TYPE=test --build-arg ENABLE_UNIFIED_COMPILATION=ON --build-arg ENABLE_TEST_TOOLS=ON .
+        docker build -t p4c --build-arg IMAGE_TYPE=test --build-arg CMAKE_UNITY_BUILD=ON --build-arg ENABLE_TEST_TOOLS=ON .
```

Comment:
It looks like this pull request may have merge conflicts with https://github.com/p4lang/p4c/pull/3770
Should we merge the tests for Ubuntu 20.04 and 22.04 first?

Reply:
Pushed. Please rebase.